decisiontree/tests/urls.py

```diff
@@ -1,11 +1,13 @@
-from django.conf.urls.defaults import *
+from django.conf.urls import include
 from django.contrib import admin
 
+
 admin.autodiscover()
 
-urlpatterns = patterns('',
+
+urlpatterns = [
     (r'^admin/', include(admin.site.urls)),
     (r'^account/', include('rapidsms.urls.login_logout')),
     (r'^scheduler/', include('rapidsms.contrib.scheduler.urls')),
```

Comment:
Sorry I didn't notice the first go-around, but 'rapidsms.contrib.scheduler' no longer exists. Looks like it was replaced by plain-old-celery, http://rapidsms.readthedocs.org/en/latest/topics/celery.html, but perhaps that change can be deferred to a future ticket.